README.md

```diff
@@ -504,3 +504,10 @@ Example:
 ```
 npm run benchmark 4 1 performance-tests/unwrapped_document_wImage.json
 ```
+
+## Running oa-cli as an executable package
```

Comment:
this commands doesnt run a cli :eyes:

Reply:
ok i change to ## Build oa-cli as an executable package?